src/Symfony/Component/Notifier/Bridge/Discord/DiscordTransport.php

```diff
@@ -66,7 +66,7 @@ protected function doSend(MessageInterface $message): SentMessage
         $content = $message->getSubject();
 
         if (\strlen($content) > 2000) {
```

Comment:
maybe time to move this to a private const?
also, we should consider using mb_strlen maybe? that'd require first checking if 2000 *UTF-8* chars are fine
last but not least, see https://support.discord.com/hc/en-us/community/posts/360031093812:
> 29/10/2020 edit: it's been kinda fixed. Now, if you type over the char limit discord will send the message as a TXT file. Not ideal but it's something.

this is worth a small investigation if anyone is up to trying with an actual Discord server

Reply:
See https://github.com/symfony/symfony/pull/39492